src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20210913161056_EES-2171_AddMethodologyNotes.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES2171_AddMethodologyNotes : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "MethodologyNotes",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    Created = table.Column<DateTime>(nullable: false),
+                    CreatedById = table.Column<Guid>(nullable: false),
```

Comment:
Would keep these columns at the end of the table, next to the `Updated*` columns as they're not as important to see

Reply:
Thanks, I've recreated the migration now and to change the column ordering.